coins.json

```diff
@@ -1404,5 +1404,32 @@
             "clientPublic": "https://api-wt2-lb.bandchain.org",
             "clientDocs": "https://docs.bandchain.org/"
         }
+    },
+    {
+        "id": "aryacoin",
+        "name": "Aryacoin",
+        "symbol": "AYA",
+        "decimals": 8,
+        "blockchain": "Bitcoin",
+        "derivationPath": "m/44'/357'/0'/0/0",
+        "curve": "secp256k1",
+        "publicKeyType": "secp256k1",
+        "p2pkhPrefix": 23,
+        "p2shPrefix": 50,
+        "publicKeyHasher": "sha256ripemd",
+        "base58Hasher": "sha256d",
+        "xpub": "xpub",
+        "xprv": "xprv",
+        "explorer": {
+            "url": "https://explorer2.aryacoin.io",
+            "txPath": "/tx/",
+            "accountPath": "/address/"
+        },
+        "info": {
+            "url": "https://aryacoin.io",
+            "client": "https://github.com/Aryacoin/Aryacoin",
+            "clientPublic": "https://explorer2.aryacoin.io",
```

Comment:
clientPublic should be the base URL of a public node RPC, if there is such.

Reply:
Updated in [2543356](https://github.com/trustwallet/wallet-core/pull/1013/commits/da7e1c62bff7d788c9c369d0d5a426c66140d283)  , the endpoint is for blockbook api v2 ,which can be used.